docker/Dockerfile

```diff
@@ -6,13 +6,32 @@ RUN apk add --update git && \
     apk add curl && \
     apk add bash && \
     apk add openjdk8 && \
-    apk add openjdk8-doc
+    apk add openjdk8-doc && \
+    apk add expect
 
 # Set sbt and java env vars
 ENV SBT_VERSION 0.13.11
 ENV SBT_HOME /usr/local/sbt
 ENV PATH ${PATH}:${SBT_HOME}/bin
 ENV PATH ${PATH}:/usr/lib/jvm/java-1.8-openjdk/bin
+ENV SCALA_VERSION=2.11.8
+ENV SCALA_HOME=/usr/share/scala
```

Comment:
I think the scala script in said RP should be turned in to a main class instead, and then invoked to do the testing. I'm not sure that we gain anything from putting a version of scala in the docker image.

In fact, we're polluting something with outside dependencies that can be resolved through `PathResolver.scala` magic.